tests/materials/processing/test_base_materials_service.py

```diff
@@ -98,8 +100,10 @@ def test_save_material_creates_powder():
         BaseMaterialService().save_material(form)
```

Comment:
There are no assertions or verifications so far. We should this add it in tests for saving. For example, since we mock the strategy, we should assert that it is called when executing save_material.